README.md

```diff
@@ -104,7 +111,9 @@ audience for the experiments (e.g. mobile users, logged-in etc).
 Experiments automatically start by default. However, a trigger function can be provided, to limit the audience or the
 page(s) where the experiment "kicks-off".
 ```javascript
-var button_color_experiment = new AlephBet.Experiment({
+import {Experiment} from "alephbet"
+
+const button_color_experiment = new Experiment({
   name: 'button color',
   trigger: function() {
```

Comment:
should we also rename the function syntax to use `=>` ?

Reply:
True, we could change this. Both would work here though.